### PR TITLE
Nov. 2020 news

### PR DESCRIPTION
**Pull request by aeslaughter**

### CODE REVIEW

**Comment by aeslaughter** [comment]
@idaholab/moose-team Please add your contributions from that past few months in the Nov. newsletter

**Comment by dschwen** [comment]
Apparently I never suggested adding the input file unit conversion to the newsletter, so if it's not too late, I'd like to write a paragraph on that.

**Comment by aeslaughter** [comment]
@GiudGiud If you review/approve again we can get this merged. Thanks for your comments.

**Review by cticenhour — DISMISSED** [review]
Short of anything else arriving last minute, this looks good to me.

**Comment by fdkong** [comment]
I will add a bit soon

**Comment by cticenhour** [comment]
Oops - thanks for chiming in @fdkong! 😄

**Comment by fdkong** [comment]
> Oops - thanks for chiming in @fdkong!

I am done. Could you proofread a couple of sentences I just added?

**Review by cticenhour — DISMISSED** [review]
Looks good!

**Comment by YaqiWang** [comment]
@aeslaughter can you give me access to your fork? I want to add a few sentences for array aux var/kernel.

**Comment by YaqiWang** [comment]
Or someone can cherry-pick this 443f4d8c7c70f0ee5031b057ff6ecdbd9fa166aa.

File: modules/doc/content/newsletter/2020_11.md
Comment (GiudGiud) [inline modules/doc/content/newsletter/2020_11.md:83]:
```suggestion
Mesh adaptivity was not done correctly when both displaced distributed mesh and exodus output were
```

**Comment by aeslaughter** [comment]
@YaqiWang Just make a PR into my branch

File: modules/doc/content/newsletter/2020_11.md
Comment (GiudGiud) [inline modules/doc/content/newsletter/2020_11.md:85]:
```suggestion
nemesis output for distributed meshes, the exodus output is useful for setting up regression tests.
```

File: modules/doc/content/newsletter/2020_11.md
Comment (GiudGiud) [inline modules/doc/content/newsletter/2020_11.md:87]:
```suggestion
the same relationship managers for both meshes.
```

File: modules/doc/content/newsletter/2020_11.md
Comment (GiudGiud) [inline modules/doc/content/newsletter/2020_11.md:17]:
```suggestion
A physical [units parsing and conversion](utils/Units.md) helper system was added to MOOSE. A unit object can be constructed form a string (e.g. `N*m`, `eV/at`, `kg*(m/s)^2`, `1/s`, `m^-3`). The unit parser supports the `*`,`/`, and `^` operators as well as parenthesis `(`, `)` to formulate composite units. Metric prefixes are supported (e.g. `MPa`, `kg`, `mm`).
```
Technically, if you support ^ (not unit exponentials, just integers) you also support -
Just a detail though, I have no opinion

File: modules/doc/content/newsletter/2020_11.md
Comment (GiudGiud) [inline modules/doc/content/newsletter/2020_11.md:19]:
```suggestion
In input files, values can be converted inline using the [brace expansion syntax](application_usage/input_syntax.md) `${units value in_unit -> out_unit}`, for example
```

**Review by GiudGiud — COMMENTED** [review]
Just a couple suggestions. 
I dont mean to hold this back so I'm not using "request changes"

The interface should allow for merging of all suggestions in a single commit btw

**Review by aeslaughter — COMMENTED** [review]
Thanks

**Comment by aeslaughter** [comment]
I reviewed and batched all the suggestions, but it doesn't seem to be added.

**Review by GiudGiud — APPROVED** [review]

**Review by cticenhour — APPROVED** [review]

**Comment by cticenhour** [comment]
@YaqiWang I did not see your post from yesterday before merging. If you want to add your commit to a PR (please ref #11447), I will get it merged.

**Pull request merged**
